apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx

```diff
@@ -56,17 +60,19 @@ export const CanvasTools = ({ publish }: CanvasToolsProps) => {
         {placementIndicator !== undefined && (
           <PlacementIndicator placement={placementIndicator} />
         )}
-      </Box>
+      </div>
     ) : null;
   }
 
   return (
-    <Box css={toolsStyle}>
+    <div className={containerStyle()}>
       <MediaBadge />
       <ResizeHandles />
-      {isPreviewMode === false && <SelectedInstanceOutline />}
-      {isPreviewMode === false && <HoveredInstanceOutline />}
+      <div className={containerStyle({ overflow: "hidden" })}>
```

Comment:
why overflow hidden as before it wasn't rendered at all?

Reply:
true